po/fr.po

```diff
@@ -19,14 +20,15 @@ msgstr ""
 "Project-Id-Version: fr\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-01-13 20:39+0100\n"
-"PO-Revision-Date: 2014-01-18 16:04-0600\n"
-"Last-Translator: Sébastien Villemot <sebastien@debian.org>\n"
-"Language-Team: \n"
+"PO-Revision-Date: 2018-10-25 13:10+0200\n"
+"Last-Translator: La Boussole <yoann@laboussole.coop>\n"
+"Language-Team: français <yoann@laboussole.coop>\n"
 "Language: French\n"
```

Comment:
According https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html
an ISO 639 two-letter language code (lowercase).
Fixed in my later commit.